python/ad-hoc-ingest/README.md

```diff
@@ -0,0 +1,66 @@
+# Ad hoc Ingest
+
+This directory contains the scripts used for ingesting the data from ad hoc sources such as hard drives that 
+The National Archives has received over the years. The scripts and process strictly applies to the data 
+which meets the following criteria:
+- The data has been already received and held on some storage media 
+- The data is NOT ingested in the existing DRI system
+- The data has been already available to the public on Discovery.
+
+## The process:
+![adhoc-ingest-flow.png](images/adhoc-ingest-flow.png)
+
+There is a script at the centre of the ingest process which drives the generic ingest. The script receives a CSV file as an input. 
+This CSV file contains the filenames, catalog reference, any other information and a checksum. The files are 
+located in a folder or other media which can be made accessible to the script. The script works chronologically n steps 
+as shown in the picture.
+
+- Read the CSV file and find out the details for each file 
+- Optionally, Get additional metadata from discovery (such as Title, Description etc.)
+- Generates an intermediate CSV file which has each row representing one metadata file. 
```

Comment:
I think this line should be on the chart as step 3 (and the "file copy / Upload" as step 4)